Formula/gloo-ctl.rb

```diff
@@ -0,0 +1,31 @@
+class GlooCtl < Formula
+  desc "Envoy-Powered API Gateway"
+  homepage "https://gloo.solo.io"
+  url "https://github.com/solo-io/gloo.git",
+      :tag      => "v0.8.3",
+      :revision => "0e749d0fd9b41f8606dca85b70f0a9981cb0d5ca"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/github.com/solo-io/gloo"
+    dir.install buildpath.children - [buildpath/".brew_home"]
+
+    cd dir do
+      # Make binary
+      system "dep", "ensure", "-vendor-only"
+      system "make", "glooctl", "TAGGED_VERSION=v0.8.3", "RELEASE=false"
+      bin.install "_output/glooctl"
+    end
+  end
+
+  test do
+    run_output = shell_output("#{bin}/glooctl 2>&1")
```

Comment:
We need a test that exercises the some of the functionality of the app.

Reply:
the `glooctl` tool depends on access to a Kubernetes cluster to do anything, so hard to make assumptions. so other than `glooctl help` and `glooctl --version`, there isn't functionality to test without access to a kubernetes cluster.

This is very similar to kubernetes-cli.eb (kubectl), and my tests are modeled after that script.

Reply:
Try to perform some operation and check that it fails, then. `kubect` is not a good example, we know its test is bad.

Reply:
I've added another test as you asked and validated it fails as expected.